pennylane_lightning/core/src/simulators/lightning_kokkos/gates/MatrixGateFunctors.hpp

```diff
@@ -23,7 +23,11 @@
 namespace {
 using namespace Pennylane::Util;
 using Kokkos::Experimental::swap;
+using Pennylane::LightningKokkos::Util::ControlBitPatterns;
+using Pennylane::LightningKokkos::Util::generateBitPatterns;
 using Pennylane::LightningKokkos::Util::one;
+using Pennylane::LightningKokkos::Util::parity_2_offset;
+using Pennylane::LightningKokkos::Util::reverseWires;
 using Pennylane::LightningKokkos::Util::vector2view;
 using Pennylane::LightningKokkos::Util::wires2Parity;
 using std::size_t;
```

Comment:
Let's not alias `std::size_t` in favour of avoiding confusion with the C `size_t` in the source code.

Reply:
updated